lib/mayaUsd/render/vp2RenderDelegate/render_delegate.h

```diff
@@ -142,6 +146,10 @@ class HdVP2RenderDelegate final : public HdRenderDelegate
 
     MHWRender::MShaderInstance* GetShaderFromCache(const TfToken& id);
```

Comment:
Since we are now having one global cache:
* why wouldn't we switch these methods to static?
* is render delegate still the right place to access the cache?